src/api/matchingHistories.ts

```diff
@@ -0,0 +1,31 @@
+import type {
+  CreateMatchingFeedbackRequest,
+  CreateMatchingFeedbackResponse,
+  MatchingHistoryResponse,
+} from 'types/matching';
+import type { SuccessResponse } from 'types/response';
+
+import { API_PATH } from 'constants/services';
+import axios from 'lib/axios';
+
+export const getRecentMatchingHistory = async () => {
+  const {
+    data: { data },
+  } = await axios.get<SuccessResponse<MatchingHistoryResponse>>(
+    API_PATH.matchingHistories.recent,
+  );
+
+  return data;
+};
+
+export const createMatchingFeedback = async (
+  payload: CreateMatchingFeedbackRequest,
+) => {
+  const { matchingHistoryId, ...feedbackForm } = payload;
+
+  const response = await axios.post<
```

Comment:
ask;
구조분해 할당하지 않고 response 전체는 반환해서 사용해야하나요?

```suggestion
  const { 
    data: { data },
  } = await axios.post<
```

Reply:
회사 및 개인 개발 시 위와 같이 반환하도록 처리하는게 습관이 되어 위와 같이 작업 했던 것 같습니다..!
구조분해 할당 처리 하였습니다.

작업 커밋: db02ded